masonry/src/text/selection.rs

```diff
@@ -283,6 +284,121 @@ impl<T: Selectable> TextWithSelection<T> {
         }
         self.layout.draw(scene, point);
     }
+
+    fn access_position_from_offset(
+        &self,
+        parent_node: &NodeBuilder,
+        offset: usize,
+        affinity: Affinity,
+    ) -> TextPosition {
+        let text = self.text().as_ref();
+        assert!(offset <= text.len());
```

Comment:
This should probably be a debug panic, or at least have an assert message.

Reply:
I believe the new version addresses this feedback.